WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
```

Comment:
I agree with the sentiment but not sure for this specific case, because when you ask "Does blog has `list_users` capability?" it doesn't make sense, because the user has the capabilities. We're actually checking to see if the current user has `list_users` (or any other) capability for this blog. Anyhow, I'll make the change, but that's kind of my take on it. Thanks!